OWNERS

```diff
@@ -3,4 +3,4 @@
 approvers:
 - deads2k
 - lavalamp
-- xuchao
+- caesarxuchao
```

Comment:
I guess this is why I can't lgtm .

Reply:
> I guess this is why I can't lgtm .

@caesarxuchao Even after this change, it won't work because you'll need to be a member of kubernetes-sigs for lgtm-ing. :upside_down_face: 

You can send a request for kubernetes-sigs membership [here](https://github.com/kubernetes/org/issues/new?template=membership.md).